### ISSUE TITLE
Generator failing

### ISSUE THREAD

**Issue by sbaechler** [jaylaw81/generator-react-sdp#2]
When I run react-jspmio it throws an exception:

```
> yo react-jspmio      
Welcome to the React JSPM.IO Component Generator
? What is the name of the project? MyApp
? What is the name of the component? NewComponent



events.js:141
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /usr/local/lib/node_modules/generator-react-jspmio/app/templates/client/base/.gitignore
    at EditionInterface.exports._copySingle (/usr/local/lib/node_modules/generator-react-jspmio/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/usr/local/lib/node_modules/generator-react-jspmio/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy.js:23:17)
    at EditionInterface.module.exports [as copyTpl] (/usr/local/lib/node_modules/generator-react-jspmio/node_modules/yeoman-generator/node_modules/mem-fs-editor/actions/copy-tpl.js:16:8)
    at template (/usr/local/lib/node_modules/generator-react-jspmio/node_modules/yeoman-generator/lib/actions/actions.js:134:11)
    at yeoman.generators.Base.extend.writing.project (/usr/local/lib/node_modules/generator-react-jspmio/app/index.js:52:9)
    at /usr/local/lib/node_modules/generator-react-jspmio/node_modules/yeoman-generator/lib/base.js:429:16
    at processImmediate [as _immediateCallback] (timers.js:374:17)
```

**Comment by sbaechler** [comment]
The .gitignore file is not present in node_modules/generator-react-jspmio/app/templates/client/base/.
When I create the file the generator works.

**Comment by tannerlyons** [comment]
Not sure of the actual problem, but this solution worked for me as a workaround:
https://github.com/Swiip/generator-gulp-angular/issues/470#issuecomment-89906624

Basically, just create the file. For some reason the .gitignore file is not being put into the directory.

**Comment by jaylaw81** [comment]
Fixed in v0.1.9. https://github.com/jaylaw81/generator-react-jspmio/releases/tag/0.1.9
